subscription/models.py

```diff
@@ -2,33 +2,34 @@
 from django.contrib.auth.models import User
 
 
-class Customer(models.Model):
-    user = models.OneToOneField(to=User, on_delete=models.CASCADE)
-    stripe_id = models.CharField(max_length=255)
-    stripe_subscription_id = models.CharField(max_length=400)
-
-    class meta:
-        verbose_name = "Customer"
-        verbose_name_plural = "Customers"
-
-    def __str__(self):
-        return f"{self.user.username}"
-
-
 class Product(models.Model):
     name = models.CharField(max_length=200)
     description = models.CharField(max_length=100, default="")
     price = models.FloatField()
     currency = models.CharField(max_length=10, default="")
-    subscription_id = models.CharField(max_length=400, default="")
+    product_id = models.CharField(max_length=200, default="")
 
-    class meta:
+    class Meta:
         verbose_name = "Product"
         verbose_name_plural = "Products"
 
+    def __str__(self):
+        return f"Price of {self.name} is {self.price} {self.currency}"
```

Comment:
Str method shouldn't be this big. Make it small! Furthermore only name should be used in str not the combination of every attribute.